Diagnostics/HealthChecker/Analyzer/Invoke-AnalyzerIISInformation.ps1

```diff
@@ -103,6 +103,7 @@ function Invoke-AnalyzerIISInformation {
 
                     if ($null -eq $cert) {
                         $problemCertList.Add("'$certHash' Doesn't exist on the server and this will cause problems.")
+                        $certHash = 'NULL'
```

Comment:
I am thinking we should change this to not be `NULL` and say `CertHash '$certHash' not found on server`. This will make it clearer where the problem is.

Reply:
![image](https://github.com/microsoft/CSS-Exchange/assets/126639605/edf85511-8dfb-40ff-abe5-5dbe436596eb)

Changes have been made and pushed the code.